spec/food_menu_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'food_menu'
+
+describe FoodMenu do
+
+  subject(:food_menu) { FoodMenu.new(food) }
+  let(:food) { {pizza: '5.00'} }
+
+  it 'contains a list of yummy dishes with their prices' do
+    expect(food_menu.food).to eq(food)
+  end
+
+  context '#print' do
+
+    it 'prints the list of yummy dishes with their prices' do
+      expect(food_menu.print).to eq("pizza: 5.00")
+    end
+
```

Comment:
Would be good to add tests that tested adding more than one dish and printing more than one dish to ensure they work as expected.